client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/Connection.spec.ts

```diff
@@ -112,10 +112,10 @@ describe("Connection", () => {
             }
         } as IHttpConnectionOptions;
 
-        var connection = new HttpConnection("http://tempuri.org", options);
+        let connection = new HttpConnection("http://tempuri.org", options);
```

Comment:
I don't mind either way, but `const` only makes the variable constant (you can't assign something else to the name), not what the variable is referencing. (See `readonly` for that.)